flink/src/main/java/org/apache/iceberg/flink/FlinkCatalog.java

```diff
@@ -684,12 +713,139 @@ public void alterPartitionColumnStatistics(ObjectPath tablePath, CatalogPartitio
 
   @Override
   public List<CatalogPartitionSpec> listPartitions(ObjectPath tablePath, CatalogPartitionSpec partitionSpec)
+      throws TableNotExistException, TableNotPartitionedException, CatalogException, PartitionSpecInvalidException {
+    Preconditions.checkNotNull(tablePath, "Table path cannot be null");
+    Preconditions.checkNotNull(partitionSpec, "CatalogPartitionSpec cannot be null");
+
+    Table table = loadIcebergTable(tablePath);
+    ensurePartitionedTable(tablePath, table);
```

Comment:
Nit:   It's more clear to leave an empty line to separate a new code block.

Reply:
Do you mean to leave an empty line between 721 and 722?